models/models.js

```diff
@@ -54,3 +54,14 @@ exports.pullAllArticles = async () => {
   }
   return result.rows;
 };
+
+exports.getArticleComments = async (id) => {
+  const result = await db.query(
+    `SELECT comment_id, votes, created_at, author, body FROM comments WHERE article_id= $1`,
+    [id]
+  );
+  if (!result.rows.length) {
+    return Promise.reject({ status: 404, msg: "No comments found" });
+  }
+  return result.rows[0];
+};
```

Comment:
You need to seperate out the result of asking for an article that doesn't exist from that of an article with no comments. At the moment they will both cause a 404 but this is only approrpriate is the article_id does not exist. If an article_id exists but has no comments, we'd expect to get a 200 back with an empty array on the key of comments. You will need to check if the article exists first in order to handle this differentiation.